src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, partnerName, sex, isMarried } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+  const personAge = age === 0 ? null : age;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      <p className="Person__age">
```

Comment:
Consider using conditional rendering here, to remove the empty block from the html
<img width="281" alt="Screenshot 2023-10-27 at 15 31 26" src="https://github.com/mate-academy/react_person/assets/135232001/b8718de8-8872-4e8e-94ba-1c472443b481">